libs/SmartSync/src/com/salesforce/androidsdk/smartsync/manager/LayoutSyncManager.java

```diff
@@ -267,18 +267,4 @@ private void initializeSoup() {
          */
         void onSyncComplete(String objectType, Layout layout);
     }
-
-    /**
-     * Enum for available data fetch modes.
-     *
-     * CACHE_ONLY - Fetches data from the cache and returns null if no data is available.
-     * CACHE_FIRST - Fetches data from the cache and falls back on the server if no data is available.
-     * SERVER_FIRST - Fetches data from the server and falls back on the cache if the server doesn't
-     * return data. The data fetched from the server is automatically cached.
-     */
-    public enum Mode {
```

Comment:
I moved this to a common file, so the cache mode can be re-used across the sync helpers.